File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        int i = 0;
        int j = 1;
        int count = 0;
        while(j < n)
        {
            if(nums[i] != nums[j])
            {
                i++;
                swap(nums[i], nums[j]);
            }
            j++;
        }
        return (i+1);
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        swap = 0
        for i in range(len(nums)):
            if nums[i] > nums[swap]:
                swap += 1
                nums[swap] = nums[i]
            i += 1
        return swap+1